### DISCUSSION TITLE
How to update to latest web-vault?

### DISCUSSION THREAD

**Discussion by basildane** [dani-garcia/vaultwarden#3208] [Q&A]
I'm running Vaultwarden 1.27.0, which came with web-vault 2022.12.0.
I'm running in Docker under compose.

I know how to update the Vaultwarden container, but I still get 2022.12.0 for the web-vault.
How can I update to the latest web-vault 2023.1.1?

**Comment by Tieneks** [comment]
As you can see [here](https://github.com/dani-garcia/vaultwarden/releases) new releases of Vaultwarden include a new version of the web-vault (if available). So, yes, the newest version of the Bitwarden web-vault is 2023.1.1, but you will have to wait for a newer version packaged with a new version of Vaultwarden.

**Reply by Tieneks** [reply]
Version 2023.1.1 will most likely be part of the next version of Vaultwarden. See [here](https://github.com/dani-garcia/vaultwarden/commit/50fc22966c43de7597960f033bde6269ea55a9ec). Hope this clears things up for you.

**Comment by BlackDex** [comment]
If you want the cutting-edge version, use the `testing` tagged images. Those contain all the latest patches currently in the `main` branch.
Which also (https://github.com/dani-garcia/vaultwarden/commit/50fc22966c43de7597960f033bde6269ea55a9ec) contains the latest web-vault.

**Comment by LiamKarlMitchell** [comment]
Can't login as outlined in #3082 I'm on 1.23
So I update docker-compose.yml to use the image name with tag testing
```vaultwarden/server:testing```

Then run these to update it.
```
docker-compose stop
docker-compose pull
docker-compose up -d
```

Now admin diagnostics tab says this and I can login again with clients.
```
Server Installed Ok 1.27.0-9366e314
Server Latest 1.27.0-9366e314
Web Installed 2023.1.1
```

Thanks
